pkg/adapter/adapter.go

```
@@ -302,30 +304,52 @@ func (a *Adapter) PollForMessages(channel *wabbit.Channel,
 	}
 }
 
+func setMsgContentType(msg wabbit.Delivery) string {
+	contentType := fmt.Sprint(msg.Headers()["Content-type"])
+	switch contentType {
+	case cloudevent.ApplicationJSON, cloudevent.TextJSON, "":
+		return cloudevent.ApplicationJSON
+	case cloudevent.ApplicationCloudEventsJSON, cloudevent.ApplicationCloudEventsBatchJSON:
+		return cloudevent.ApplicationCloudEventsJSON
+	default:
+		return contentType
```

I wonder if this should return an error if there's no contenttype?